src/AWS/CRT/Auth/SigningResult.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+namespace AWS\CRT\Auth;
+
+use AWS\CRT\NativeResource;
+use AWS\CRT\HTTP\Request;
+
+class SigningResult extends NativeResource {
+    protected function __construct($native) {
+        parent::__construct();
+
+        $this->acquire($native);
+    }
+
+    function __destruct() {
+        // No release necessary, SigningResults are transient
```

Comment:
Why is this comment here?  And if it's not necessary, why do you do it on the next line?

Reply:
Fixed, this was indeed confusing as hell